src/commands/new/test.ts

```diff
@@ -12,27 +14,33 @@ module.exports = {
       print,
       parameters,
       template: { generate },
+      utils,
       args
     } = toolbox;
 
     try {
       const options = args.parseOptions(TEST_OPTIONS);
       const outputDirectory = args.getOutputDirectory();
+      const filters = JSON.parse(parameters.options.filters) || [];
```

Comment:
I was incorrect earlier. This is how to use short-circuit evaluation without a JSON parse error.
```suggestion
      const filters = JSON.parse(parameters.options.filters || '[]');
```

Reply:
Done